examples/multiple_mzr_clusters/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = ">= 1.60.0"
+      version = "1.62.0"
```

Comment:
This example is purposely using `>=` to test latest provider. There are other examples locked into lowest supported version